pkg/receiver/smartagentreceiver/output.go

```diff
@@ -208,6 +209,41 @@ func (out *output) Copy() types.Output {
 	return &cp
 }
 
+func (out *output) SendMetrics(metrics ...pmetric.Metric) {
```

Comment:
Is this function wired to the code? I don't see it being used... It's unclear what the scope of the PR is. If this only adds filtering capability without using it, this PR must be unrelated I guess.

Reply:
It's not used yet, this PR is a smaller take on https://github.com/signalfx/splunk-otel-collector/pull/4243 which wires this function into the prometheus monitor.